src/tests/RnR_tests/RnR.test.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { mount, configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import App from '../../components/app';
+import { App } from '../../components/app';
```

Comment:
@shodgetts this imports unconnected version of app for tests
@jbc5099 thanks for fixing this